packages/block-library/src/file/inspector.js

```diff
@@ -68,20 +68,18 @@ export default function FileBlockInspector( {
 						) }
 					</PanelBody>
 				) }
-				<PanelBody title={ __( 'Text link settings' ) }>
+				<PanelBody title={ __( 'Settings' ) }>
 					<SelectControl
 						label={ __( 'Link to' ) }
 						value={ textLinkHref }
 						options={ linkDestinationOptions }
 						onChange={ changeLinkDestinationOption }
 					/>
 					<ToggleControl
-						label={ __( 'Open in new tab' ) }
+						label={ __( 'Open link in new tab' ) }
```

Comment:
I think we should leave this as is. We use `Open in new tab` in all other instance of similar controls for this attribute.

Reply:
Good call!